bounties.md

```diff
@@ -26,5 +26,13 @@ We are at the start of building out more resources for contributors. If you are
 Additionally, feel free to attend Engineering Stand-up Every Weekday 17:00 UTC in [Discord](https://discord.gg/shapeshift) #Standup-AllAccess. You can introduce yourself, and we can often find someone to pair up with you if you need assistance for a few minutes after the meeting.
 
 #### Looking to integrate a large feature?
```

Comment:
We don't lint for this yet, but I'm trying to instigate best practice markdown use across all of our markdown files - add a new line after any heading.